vault/resource_aws_secret_backend.go

```diff
@@ -98,6 +98,13 @@ func awsSecretBackendResource() *schema.Resource {
 				Computed:    true,
 				Description: "Template describing how dynamic usernames are generated.",
 			},
+			"local": {
```

Comment:
Could we also add a check in the test file [here](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_aws_secret_backend_test.go#L17) that ensures the fields default works as expected? We can add the following check:

```
resource.TestCheckResourceAttr(resourceName, "local", "false"),
```